core/ui.libs/src/utils/profileUtils.ts

```diff
@@ -42,29 +45,41 @@ export async function getUserAlias(
   accountId: string = runtime.activeAccount,
   accountDetails?: any,
 ): Promise<string> {
-  const otherProfile = new Profile({
-    ...(runtime as any),
-    profileOwner: accountId,
-    accountId: runtime.activeAccount,
-  });
-  let details = { ...accountDetails };
+  const cacheID = `${runtime.activeAccount}.${accountId}`;
 
-  try {
-    details = cloneDeep(lodash, await otherProfile.getProfileProperty('accountDetails') || { });
-  } catch (ex) {
-    // no permissions
-  }
+  if (!aliasCache[cacheID]) {
+    aliasCache[cacheID] = (async () => {
+      const otherProfile = new Profile({
+        ...(runtime as any),
```

Comment:
as bcc.Runtime ?

Reply:
This will cause an error, because not all properties of the runtime are supported within the `ProfileOptions`